test/test.js

```diff
@@ -13,6 +13,7 @@ describe('DynamoDBStore', function () {
     describe('Instantiation', function () {
         it('should be able to be created', function () {
             var store = new DynamoDBStore({
+                ttl: false,
```

Comment:
Thanks! Sadly no ETA as our app went to production shortly after I started doing this PR. That has taken all of my time.

I know this has been a little while since I touched it. I wanted to test it out with our application before wrapping it up. It should be pretty close. I have also been thinking about how I can allow existing customers to start using this functionality.